gulpfile.babel.js

```diff
@@ -0,0 +1,43 @@
+import gulp from 'gulp';
+import nodemon from 'gulp-nodemon';
+import mocha from 'gulp-mocha';
+import concat from 'gulp-concat';
+import uglify from 'gulp-uglify';
+import rename from 'gulp-rename';
+import sass from 'gulp-ruby-sass';
+import imagemin from 'gulp-imagemin';
+import cache from 'gulp-cache';
+
+gulp.task('scripts', () => gulp.src(['public/js/**', 'app/**/*.js'])
```

Comment:
_scripts_ should run on only client files i.e. _public/**/*.js_

Reply:
ok, will effect this changes